deploy/compose/docker-compose.yml

```diff
@@ -125,23 +115,18 @@ services:
   tink-server:
     image: ${TINK_SERVER_IMAGE}
     environment:
-      FACILITY: ${FACILITY:-onprem}
-      PACKET_ENV: ${PACKET_ENV:-testing}
-      PACKET_VERSION: ${PACKET_VERSION:-ignored}
+      FACILITY: $FACILITY
       PGDATABASE: tinkerbell
       PGHOST: db
       PGPASSWORD: tinkerbell
       PGPORT: 5432
       PGSSLMODE: disable
       PGUSER: tinkerbell
-      ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-1}
-      ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-ignored}
-      TINK_AUTH_PASSWORD: ${TINKERBELL_TINK_PASSWORD:-admin}
-      TINK_AUTH_USERNAME: ${TINKERBELL_TINK_USERNAME:-admin}
       TINKERBELL_GRPC_AUTHORITY: :42113
       TINKERBELL_HTTP_AUTHORITY: :42114
+      TINKERBELL_TLS: $TINKERBELL_TLS
```

Comment:
some of the env vars are in this format `$VAR` and some are in `${VAR}`. do we want to just use one format for all?

Reply:
Hmm yep I think I meant to do that too but wanted it as a separate commit and then just forgot about it. Let me give it a try without the `{}`.

Reply:
Updated to remove the `{}`s and made sure it all still works via the vagrant provider. PTAL again.